src/services/AttachmentResolver.js

```diff
@@ -43,190 +45,64 @@ export default class AttachmentResolver {
 		this.#user = user
 		this.#shareToken = shareToken
 		this.#currentDirectory = currentDirectory
-		fileId ||= session?.documentId
-		this.#attachmentDirectory = `.attachments.${fileId}`
+		this.#documentId = fileId ?? session?.documentId
```

Comment:
Since we do no longer initialize the resolver without fileId, i think we can skip the second part and rather throw if no file id was passed?

Reply:
So far, we don't pass the fileId from `Editor.vue` yet as we already pass the `session` there, which includes the `fileId`. We could change this though and always pass in the `fileId`. In any case we can remove the `?` after `session` because we need a `fileId` - either directly provided or from the provided session.

Reply:
All right, then let's stick with that

Reply:
Also done now.